### ISSUE TITLE
Add new reference quantized SSD-MobileNet model

### ISSUE THREAD

**Issue by psyhtest** [mlperf/inference_policies#49]
Following WG's discussion on 14/Aug/2019, I suggest we document the request and decision here.

@DilipSequeira:
> we would like the symmetrically quantized model for MobiletNet-SSD that Habana trained to be added to the list of allowed models...

**Comment by psyhtest** [comment]
The current reference quantized SSD-MobileNet is from Habana:

| model | framework | accuracy | dataset | model link | model source | precision | notes |
| ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- |
| ssd-mobilenet 300x300 quantized finetuned | tensorflow | mAP 0.23594 | coco resized to 300x300 | [from zenodo](https://zenodo.org/record/3252084/files/mobilenet_v1_ssd_8bit_finetuned.tar.gz) | Habana | int8 | ??? |

Unfortunately, I can't find any reference to the new model in my email.

@itayhubara, could you please ask your colleague to upload it to the same Zenodo record? Or, if a new record needs to be created, to upload both the old and the new models to the same record under descriptive names (e.g. `ssd_mobilenet_v1_coco_quantized_symmetric.tar.gz` and `ssd_mobilenet_v1_coco_quantized_asymmetric.tar.gz`)?

**Comment by jveejay** [comment]
Yea, I dug through my emails as well and I can't find it.

**Comment by christ1ne** [comment]
added the model
